docs/source/environment_variables.rst

```diff
@@ -0,0 +1,664 @@
+
+Values accepted
+^^^^^^^^^^^^^^^
+The default value is ``false``
+
+ONEFLOW_KERNEL_DISABLE_BLOB_ACCESS_CHECKER
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+Only use blob_access_checker after enabling, because blob_access_checker is for correctness assurance, and closing it in some cases can increase the kernel overhead.
+
+Values accepted
+^^^^^^^^^^^^^^^
+The default value is ``true``
+
+ONEFLOW_ONE_EMBEDDING_EMBEDDING_SHUFFLE
+_INDEPENTENT_STREAM
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+Whether to put the EmbeddingShuffle of the OneEmbedding module on a separate stream for overlapping execution.
+
+Values accepted
+^^^^^^^^^^^^^^^
+The default value is ``false``
+
+ONEFLOW_ONE_EMBEDDING_GRADIENT_SHUFFLE_USE_FP16
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+Whether to allow the EmbeddingGradientShuffle operator of the OneEmbedding module to use the FP16 data type in the AMP case.
+
+Values accepted
+^^^^^^^^^^^^^^^
+The default value is ``true``
+
+ONEFLOW_ONE_EMBEDDING_NOT_FUSE_CAST_TO_UPDATE
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+Whether to disable the fusion of cast type conversion and parameter update of OneEmbedding parameters into one operator in the case of AMP
+
+Values accepted
+^^^^^^^^^^^^^^^
+The default value is ``false``
+
+ONEFLOW_DEBUG_KERNEL_SYNC_CHECK_NUMERICS_DUMP
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+When the value appears NaN or Inf, save the data Dump.
+
+Values accepted
+^^^^^^^^^^^^^^^
+The default value is ``false``
+
+ONEFLOW_DECODER_ENABLE_NVJPEG_HARDWARE_ACCELERATION
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+``NVJPEG_VER_MAJOR`` should be bigger than 11. It can accelerate nvjpeg hardware, warm up jpeg decoder and hw_jpeg decoder, `#5851 <https://github.com/Oneflow-Inc/oneflow/pull/5851>`_. 
```

Comment:
```suggestion
``NVJPEG_VER_MAJOR`` should be bigger than ``11``. It can accelerate nvjpeg hardware, warm up jpeg decoder and hw_jpeg decoder, `#5851 <https://github.com/Oneflow-Inc/oneflow/pull/5851>`_. 
```